cl/assets/templates/base.html

```diff
@@ -477,9 +477,10 @@ <h1>You did not supply the "private" variable to your template.
     _paq.push(['trackPageView']);
     _paq.push(['enableLinkTracking']);
     (function () {
-      var u = "//matomo.courtlistener.com/";
+      var u = "{{ MATOMO_BASE_URL }}";
```

Comment:
I just noticed that there's an issue with the way this is working. In the settings, `MATOMO_BASE_URL` represents the *internal* URL of MATOMO, within the CourtListener network. It does *not* represent the external URL. You can see how we use it, for example, here: 

https://github.com/freelawproject/courtlistener/blob/e389c613a1ec8a9cd5cd6afbb9fb0e0b4e1b467a/cl/lib/decorators.py#L108-L122

So I think either we need to do some better DNS work inside our network (we don't do any now, but maybe this isn't crazy), or we need a setting for Internal and another for external. 

What do you think?

Reply:
I've updated this. Now there is now an additional setting for frontend: `MATOMO_FRONTEND_BASE_URL = "//matomo.courtlistener.com/"`